testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -11954,6 +11955,7 @@ void assertSubFunReturns(boolean binary, String s, int start,
         "Invalid argument '': the length of the string describing "
             + "the trimmed character must be 1",
         true);
+    f.checkFails("trim()", "Invalid number of arguments to function 'TRIM'. Was expecting at least 2 arguments", false);
```

Comment:
This test cannot pass...

Reply:
then the PR is incomplete

Reply:
Yes, give me some time and I'll see if I can handle it.

Reply:
Please remove the LGTM tag before I fix it.